Sources/KlaviyoCore/Models/PushEnablement.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-public enum PushEnablement: String, Codable {
+public enum PushEnablement: String, Codable, Sendable {
```

Comment:
Does adding all of these imply that we didn't have concurrency protection in previous versions? Or was this also necessary to add with the TCA updates.

Reply:
I think actually other than the bug that was caught by an outside developer we were being pretty careful with concurrency. For a lot of these changes we are just making the compiler happy but hopefully it will stop us from making future concurrency mistakes.